package.json

```diff
@@ -9,7 +9,7 @@
     "dist/**/*"
   ],
   "repository": "https://github.com/geolonia/embed-react",
-  "author": "Keitaroh Kobayashi <k.kobayashi@geolonia.com>",
+  "author": "Geolonia Dev Team <dev@geolonia.com>",
```

Comment:
![review:In My Opinion](https://img.shields.io/badge/review-imo-yellow.svg)

In my opinion, we don't have to expose our email address since we have GitHub Issues to receive public feedback.

```suggestion
  "author": "Geolonia (https://geolonia.com)",
```